### ISSUE TITLE
Question to etc development team

### ISSUE THREAD

**Issue by etcnoob** [ethereumclassic/faq#7]
Hi,

I buyed ETH in stead of ETC. 

Is it possible to get non-random walletnumber? This way I could get my ETC credits. It is very stupid of me of course. But it happend.

Kind regards

**Comment by cseberino** [comment]
Mathemetically impossible to find private key to a random address.  Sorry.

cs

_______________________________________

Christian Seberino, Ph.D.
Phone: (936) 828-8747
Email: cseberino@gmail.com
_______________________________________

**Comment by etcnoob** [comment]
Hi cseberino,

Many thanks for your reaction/response.

I have the private numbers of course, where I sent the coin (I know it's not much, but for me it's a lot of money).

But if I have the key, would it be possible to get the coin back (or sent to my real address)?

kindly regards

**Comment by splix** [comment]
As you have the private key, then you already have access to the wallet. Just use it

**Comment by etcnoob** [comment]
Splix, 

many thanks for your reaction. I really appreciate your answer(s).

I am a little bit confused, because I just lost my coin.

I said it wrong, I hope you understand a little bit. What I mean is that I have the wallet id, but the private keys I don't have it. And I am certain that the wallet where I sent the coin to, is not being used by a person. Although the wallet does exist.

If there is any hope, do you know where I should inform?

**Comment by splix** [comment]
Hm, from your first message I understood it as you've used ETH address instead of ETC address. If that's the problem, you can get your ETH from that address. 
But if you just used a random address (how that's possible?) then there're no way to return them

**Comment by etcnoob** [comment]
Almost. I used the ETC address in stead of the ETH address.

I got all wallet-numbers, and the private key (of my ETH address).

So you saying that the private keys for ETC and ETH are the same? In that case I will easily get my coin back. Did I understand you correctly?

**Comment by splix** [comment]
Yes, same. It's Private Key for Ethereum address in general, it works for both ETH and ETC chains

**Comment by cseberino** [comment]
Bottom line is whatever ETC address you sent your ETC to must have a
private key you have access to.
Nothing else will do.   And yes the math is the same for ETH and ETC.   If
you have the private key you are good.

cs

_______________________________________

Christian Seberino, Ph.D.
Phone: (936) 828-8747
Email: cseberino@gmail.com
_______________________________________

**Comment by etcnoob** [comment]
Omg I made a big mistake, of course I am not thinking normally atm.

I ordered one ETH_coin on an exchange site and filled in my ETC wallet number in the ETH (sent_to_address) field, but the exchange site recognized this as an ETH-walletnumber. I was thinking till now, that the exchange site would have sent the coin to ETC, but it is sended to a non-existing ETH walletnumber. Or can I still access this ETH_number?

Maybe it's better for me to sleep and go on further tomorrow. But the information in this/last posts is correctly.

The website where I have my wallets, provides all coins. And I have 1 ETC wallet and 1 ETH wallet, but the numbers are differently.

**Comment by etcnoob** [comment]
Case solved.

Many many many thanks cseberino and splix.

I didn't knew that etc and eth are the same (technicallly)

**Comment by splix** [comment]
Exactly same. Just two different chains of same Ethereum. Even had same block history, till Jul 20th, 2016

**Comment by gitr0n1n** [comment]
This issue is resolved. Can be closed. @realcodywburns
